cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -11,6 +11,7 @@
 import java.nio.file.StandardOpenOption;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
```

Comment:
I think Pair/Stream imports are no longer used

Reply:
Done in [9458940](https://github.com/project-ncl/gradle-manipulator/pull/385/commits/945894036cc17e2f333fcb61f5e574ad27792d83).